api/src/main/java/com/xwiki/googleapps/GoogleAppsScriptService.java

```diff
@@ -74,134 +73,141 @@ public boolean isActive() throws XWikiException {
      * @since 3.0
      */
     @Unstable
-    public boolean useDrive() {
-        return manager.useDrive();
+    public boolean isDriveEnabled()
+    {
+        return manager.isDriveEnabled();
     }
 
-
     /**
-     * Reads the manifest to find when the JAR file was assembled by maven.
+     * Finds when the JAR file was assembled by maven.
+     *
      * @return the build date.
      * @since 3.0
      */
     @Unstable
-    public Date getBuildTime() {
+    public Date getBuildTime()
+    {
         return manager.getBuildTime();
     }
 
-    /** Inspects the stored information to see if an authorization or a redirect needs to be pronounced.
+    /**
+     * Finds the version of the JAR file.
+     *
+     * @return the build version.
+     * @since 3.0
+     */
+    @Unstable
+    public String getBuildVersion() { return manager.getBuildVersion(); }
+
+    /**
+     * Inspects the stored information to see if an authorization or a redirect needs to be pronounced.
      *
      * @return found credential
-     * @throws XWikiException if the interaction with xwiki failed
-     * @throws IOException if a communication problem to Google services occured
      * @since 3.0
      */
     @Unstable
-    public Credential authorize() throws XWikiException, IOException {
+    public Credential authorize()
+    {
         return manager.authorize();
```

Comment:
This can throw an exception. Look for script service examples in platform to see how they handle exceptions.